frame/vesting/src/weights.rs

```diff
@@ -48,8 +47,12 @@ pub trait WeightInfo {
 	fn vest_unlocked(l: u32, ) -> Weight;
 	fn vest_other_locked(l: u32, ) -> Weight;
 	fn vest_other_unlocked(l: u32, ) -> Weight;
-	fn vested_transfer(l: u32, ) -> Weight;
-	fn force_vested_transfer(l: u32, ) -> Weight;
+	fn first_vested_transfer(l: u32, ) -> Weight;
+	fn last_vested_transfer(l: u32, ) -> Weight;
+	fn first_force_vested_transfer(l: u32) -> Weight;
+	fn last_force_vested_transfer(l: u32) -> Weight;
+	fn unlocking_merge_schedules(l: u32) -> Weight;
+	fn not_unlocking_merge_schedules(l: u32) -> Weight;
 }
```

Comment:
Manually added these to compile under the assumption this file will be correctly updated when I run weights with CI